applications/dashboard/controllers/api/MediaApiController.php

```diff
@@ -111,6 +115,13 @@ private function doUpload(UploadedFile $upload, $type) {
                 }
         }
 
+        // image dimensions are higher than limit, it needs resizing
+        if ($this->config->get("ImageUpload.Limits.Enabled")) {
+            if ($media['ImageWidth'] > c("ImageUpload.Limits.Width") || $media['ImageHeight'] > c("ImageUpload.Limits.Height")) {
```

Comment:
```suggestion
            if ($media['ImageWidth'] > $this->config->get("ImageUpload.Limits.Width") || $media['ImageHeight'] > $this->config->get("ImageUpload.Limits.Height")) {
```

You got 1 and not the others.